src/Map/src/Bridge/Google/.gitattributes

```diff
@@ -1,4 +1,8 @@
-/Tests export-ignore
-/phpunit.xml.dist export-ignore
+/assets/src export-ignore
+/assets/test export-ignore
+/assets/vitest.config.js export-ignore
+/tests export-ignore
 /.gitattributes export-ignore
 /.gitignore export-ignore
+/.symfony.bundle.yaml export-ignore
```

Comment:
There is no `.symfony.bundle.yaml`
<img width="394" alt="image" src="https://github.com/user-attachments/assets/0280ca6b-b1c2-4696-ae3b-52b7793db293">